docs/source/advanced_features/cache.rst

```diff
@@ -1,3 +1,6 @@
+..
+ Copyright 2021 Graviti. Licensed under MIT License.
+ 
```

Comment:
do we need a blank line after the copyright header? @graczhual

Reply:
I haven't found any rules or requirements. But in the format of our python files, we can leave one line blank.